e2e-tests/monitoring/02-assert.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 300
+---
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: monitoring-mysql
+spec:
+  template:
+    spec:
+      containers:
+      - name: mysql
+      - name: pmm-client
+status:
+  observedGeneration: 1
+  replicas: 3
+  readyReplicas: 3
+  currentReplicas: 3
+  updatedReplicas: 3
+  collisionCount: 0
+---
+kind: StatefulSet
+apiVersion: apps/v1
```

Comment:
I guess we need CR with desired state here also

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/21/commits/757e92c61cb0691b3713cd46d05a7091094ec2e7